src/components/AnonymousChat/child/ChatItemMyTextV2.tsx

```diff
@@ -152,7 +238,9 @@ const ChatItemMyTextV2 = ({
                 />
 
                 <View style={{flexDirection: 'row'}}>
-                  <Text style={messageStyle(true, messageType)}>{message}</Text>
+                  {attachments.length <= 0 && (
+                    <Text style={messageStyle(true, messageType)}>{message}</Text>
+                  )}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The message text is now only rendered if there are no attachments. Ensure that this is the desired behavior and that it doesn't cause any issues with messages that should include both text and attachments.


<!-- This is an auto-generated comment by OSS CodeRabbit -->